SRS.md

```diff
@@ -244,13 +220,9 @@ The following user interfaces are implemented:
 
 Dashboard - lists all current exams. Here it is possible to create new exams.
 
-Create/ Edit Exam - provides the fields for a new exam. Also, it should be possible to edit the information of the selected exam here and to navigate to the creation interface of a topic. 
-
-Create/ Edit Topic - provides the fields for a new topic. It should be possible to edit the information of the selected topic.
-
-View Exam - shows detailed information about an exam and a list of the existing topics. It is possible here to navigate to edit exam.
+Create/ Edit Exam - provides the fields for a new exam. Also, it should be possible to edit the information of the selected exam here. 
 
-View Topic - shows detailed information about a topic and contains a timer for the selected exam. Here it is posible to navigate to edit topic and start the timer.
+View Exam - shows detailed information about an exam and the estimated time. Also provides the oppurtunity to start and stop the time for the estimated tim. It is possible here to navigate to edit exam.
```

Comment:
shows detailed information about an exam and the estimated time. Also, it provides the opportunity to start and stop the timer for the suggested time. It is possible here to navigate to edit exam.